cpp/collections_module/collections.cpp

```diff
@@ -1,9 +1,23 @@
 #include <mgp.hpp>
 
+#include "algorithms/collections.hpp"
+
+const char *kProcedureUnionALl = "unionAll";
+
+const char *kReturnUnionAll="return_list";
+
+
+const char *kArgumentList1= "list1";
```

Comment:
use `constexpr std::string:view` instead of `const char*` as it can be changed for better code style

Reply:
I get an error, same as Idora in Slack